src/extension.ts

```diff
@@ -41,6 +41,7 @@ import * as fs from "fs";
 import * as semver from "semver";
 import { getJavaHome } from "./getJavaHome";
 import { getJavaOptions } from "./getJavaOptions";
+import * as scalaDebugger from "./debugger";
```

Comment:
Can we use the name `debugger` instead? I prefer to avoid new names unless it's necessary (conflict with another `debugger` for example).

Reply:
we cannot use the `debugger` name. when trying to do so I get error `Identifier expected.ts(1003)`

Reply:
Hmm, it looks like `debugger` is a JavaScript keyword https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/debugger

Can we rename the file to `scalaDebugger` in that case? I think it's good to try and keep the variable name consistent with the imported filename.